src/components/Info/Info.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const InfoSection = props => (
+  <section className={`bx--row ${props.className} info-section`}>
+    <div className="bx--col-md-8 bx--col-lg-4 bx--col-xlg-3">
+      <h3 className="info-section__heading">{props.heading}</h3>
+    </div>
+    {props.children}
+  </section>
+);
+
+const InfoCard = props => {
+  const splitHeading = createArrayFromPhrase(props.heading);
+
+  return (
+    <article className="info-card bx--col-md-4 bx--col-lg-4 bx--col-xlg-3 bx--offset-xlg-1">
```

Comment:
The `<article>` tag may semantically wrong for accessibility, see https://www.carbondesignsystem.com/tutorial/react-step-4#check-accessibility,